### ISSUE TITLE
Faster startup

### ISSUE THREAD

**Issue by PallHaraldsson** [Gnimuc/CImGui.jl#33]
It's already rather fast for it and the Plots package based on it. What's possible:

```
$ julia -O1 --compile=min -q
julia> @time using CImGui
  0.510211 seconds (598.01 k allocations: 42.734 MiB)

vs. on defaults

julia> @time using CImGui
  1.119417 seconds (705.33 k allocations: 48.456 MiB, 0.73% gc time)
```

I'm looking into a PR to get this as a default, I'm just not sure you would want it. At least not with compile-min.

**Comment by PallHaraldsson** [comment]
Actually the PR would need to be at:

```
$ julia -O1 --compile=min -q
julia> @time using GLFW
  0.438421 seconds (472.77 k allocations: 33.463 MiB)

julia> 
pharaldsson_sym@SYMLINUX011:~/nowcasting/eagdpsmall_jl$ julia -q
julia> @time using GLFW
  1.011249 seconds (579.78 k allocations: 39.166 MiB)
```

Do you know, would people use this package in a tight loop? I mean I forrsee you want graphics in such a loop, just not sure if it applies to that package and/or this package (and/or the Plotting on based on your).

**Comment by Gnimuc** [comment]
Just out of curiosity. Has this been improved with Julia 1.6-rc1?

**Comment by Gnimuc** [comment]
I think this has already been fixed on Julia 1.6.
